rpc/message.go

```diff
@@ -268,29 +269,147 @@ type CmdArgs interface {
 	cmdArgs()
 }
 
-type Event string
+type Event struct {
+	EventType
+	CmdArgs
+}
+
+func BuildIncomingEvent(eventType EventType, data MessageData) (Event, error) {
```

Comment:
```suggestion
func UnmarshalEvent(eventType EventType, data MessageData) (Event, error) {
```

Reply:
fixed in [4761252](https://github.com/dennisrijsdijk/disgo/commit/47612524627647d77f406eb356b1f85dfa952f1e)